src/detect-pcre.c

```diff
@@ -647,29 +653,38 @@ static DetectPcreData *DetectPcreParse (DetectEngineCtx *de_ctx,
                 "Falling back to regular PCRE handling (%s:%d)",
                 regexstr, de_ctx->rule_file, de_ctx->rule_line);
     }
+
+    if (jit) {
+        pd->parse_regex.jit_stack = pcre_jit_stack_alloc(32*1024, 512*124);
+        if (pd->parse_regex.jit_stack == NULL) {
+            SCLogError(SC_ERR_PCRE_JITSTACK, "pcre jit stack failed");
```

Comment:
should this be an error or should we fall back to non-jit in this case?

Reply:
Good point. In the rare case of an error, I'll emit a warning message and fall back.